File: problems/1295.find-numbers-with-even-number-of-digits/findNumbersWithEvenNumberOfDigits.js
/**
 * 1295. Find Numbers with Even Number of Digits
 * @param {number[]} nums
 * @return {number}
 */
const findNumbers = function (nums) {
  return nums.filter((num) => num.toString().length % 2 === 0).length;
};

module.exports = findNumbers;

File: problems/1295.find-numbers-with-even-number-of-digits/find_numbers_with_even_number_of_digits.py
"""Find Numbers with Even Number of Digits"""
from ast import List


class FindNumbers:
    """
    Class to find the count of numbers in the given list that have an even number of digits.
    """
    def find_numbers(self, nums: List[int]) -> int:
        """
        Finds the count of numbers in the given list that have an even number of digits.
        
        Args:
            nums (List[int]): A list of integers.
            
        Returns:
            int: The count of numbers with an even number of digits.
        """
        count = 0
        for num in nums:
            if len(str(num)) % 2 == 0:
                count += 1
        return count
